src/native/libs/System.Security.Cryptography.Native/pal_evp_pkey_ml_dsa.c

```diff
@@ -43,6 +43,8 @@ int32_t CryptoNative_MLDsaGetPalId(const EVP_PKEY* pKey, int32_t* mldsaId, int32
 #endif
 
     (void)pKey;
+    (void)hasSeed;
+    (void)hasSecretKey;
```

Comment:
Actually, these should probably be

```suggestion
    *hasSeed = 0;
    *hasSecretKey = 0;
```